docker-compose.yml

```diff
@@ -13,28 +13,8 @@ services:
       - NODE_ID=24
     ports:
       - 11211:11211
-  regular01:
```

Comment:
We don't want to keep the 'old' docker-compose?

Reply:
Well I was using docker-compose to run multiple nodes on the same time, but was afraid that concurrent joins cause issues

Reply:
Yeah okay, I thought maybe have it next to this new docker-compose. But both ways are fine.